lib/modules/dosomething/dosomething_fact/dosomething_fact.module

```diff
@@ -75,6 +75,10 @@ function dosomething_fact_get_facts_data($node, $field_names) {
     if ($field_data) {
       if (array_key_exists($language, $field_data)) {
         $entity = dosomething_fact_get_entity($field_data, $language);
+       } else {
+        //if we do not have a translation for the current user's language, get language of user that created the reportback
+        $alpha_language = $node->language;
```

Comment:
could do this without the temp variable and just call the function with `$node->language`